src/main/java/ch/njol/skript/effects/EffExit.java

```diff
@@ -104,11 +104,19 @@ private static int numLevels(final int type) {
 			return ScriptLoader.currentSections.size();
 		int r = 0;
 		for (final TriggerSection s : ScriptLoader.currentSections) {
-			if (type == CONDITIONALS ? s instanceof Conditional : s instanceof Loop || s instanceof While)
+			if (type == CONDITIONALS && s instanceof ConditionalSection)
```

Comment:
Do you need to determine if it's a conditional and a section of a conditional? Seems redundant.

Reply:
I clicked single comment but it came up as a review, rip.